docs/optlib.rst

```diff
@@ -1,53 +1,75 @@
 .. _optlib:
 
-Extending ctags with Regex parser(*optlib*)
+Extending ctags with Regex parser (*optlib*)
 ---------------------------------------------------------------------
 
 :Maintainer: Masatake YAMATO <yamato@redhat.com>
 
+.. TODO:
+	review extras, fields, and roles sections
+	possibly restructure this file's section ordering
+	add documentation for --_mtable-extend-<LANG>
+	add documentation for tjump, treset, tquit flags
+	add a section on debugging
+	add a section on langdef base parser flag, including
+		shared/dedicated/bidirectional directions
+
 ----
 
-.. NOT REVIEWED
+.. Q: shouldn't the section about option files (preload especially) go in
+	their own section somewhere else in the docs? They're not specifically
+	for "Extending ctags" - they can be used for any command options that
+	you want to use permanently. It's really the new language parsers using
+	--regex-<LANG> and such that are about "Extending ctags", no?
+
 
 Option files
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Option file is a file in which command line options are written line
-by line. ctags loads it and runs as if the options in the file are
+An "option" file is a file in which command line options are written line
+by line. ``ctags`` loads it and runs as if the options in the file were
 passed in command line.
 
 Following file is an example of option file.
 
-.. code-block::
+.. code-block:: python
 
 	# Exclude directories that don't contain real code
 	--exclude=Units
+		# indentation is ignored
 		--exclude=tinst-root
-		--exclude=Tmain
+	--exclude=Tmain
 
 `#` can be used as a start marker of a line comment.
 Whitespaces at the start of lines are ignored during loading.
 
-Preload and optlib are the category of option files.
+There are two categories of option files, though they both contain command
+line options: **preload** and **optlib** option files.
+
+.. Q: do we really want to call the non-preload option files "optlib"?
+	That name seems like an internal detail. Users of ctags never see that
+	name anywhere except in these docs, and it's weird. How about
+	"specified" option files, or "requested" or some such? (i.e., the file
+	is explicitly specified or requested when ctags is run)
```

Comment:
I agree. 

The idea behind the term "optlib" is written at the head of man/ctags-optlib.7.rst.in.